### ISSUE TITLE
Unify naming of chart values

### ISSUE THREAD

**Issue by ableuler** [SwissDataScienceCenter/renku#635]
We are currently lacking consistency in the naming of things throughout our charts. This leads to the same concepts being given different names. 

- [ ] URLs: We use chart values to pass URLs of other services into our deployments. However, what's called `someserviceUrl` in one place can be called `someserviceHost` in another place, and it is sometimes unclear if a full URL or only a <scheme>://<host-name> is expected in the case of path-based routing. 
To solve this we should use full service URLs as values in the charts and label them as `servicenameUrl`. If the application expects only parts of the URL to be passed in, use helm helpers to split the URL into its parts. See for example https://github.com/SwissDataScienceCenter/renku-gateway/pull/142/files.

- [ ] OAuth: Renku builds on multiple OAuth providers and even more clients. The corresponding secrets and ids should be name following a consistent pattern, for example `<provider-name><Client-name>Client<Secret/Id>` which would result in `gitlabRenkuClientId` or `keycloakGitlabClientSecret`.

- there may be other cases of inconsistent naming...?

**Comment by rokroskar** [comment]
Is there a PR for this already?

**Comment by ableuler** [comment]
The URL aspect of this was already partially addressed in my aborted chart refactoring attempt a few months ago. For some of that there are stale PRs which should be brought back to life:
- https://github.com/SwissDataScienceCenter/renku-notebooks/pull/165
- https://github.com/SwissDataScienceCenter/renku-gateway/pull/142

**Comment by rokroskar** [comment]
Do you think you'll have the cycles to do it this sprint?

**Comment by rokroskar** [comment]
Is this still something we want to do?

**Comment by ableuler** [comment]
> Is this still something we want to do?

The possibility to modify the the manifest files after template rendering with pulumi reduces the need for consistent naming throughout the component charts. I guess it would still make dev life a bit easier, but we can have consistent naming of things on the global Renku level even if the components don't follow suit. So my answer would be "yes", but with lower priority.

**Comment by rokroskar** [comment]
The naming changes proposed above make sense and I think are a bit pulumi/helm-independent. It also doesn't seem like a lot of work, but if we're not going to do it I would rather just close this.

**Comment by ableuler** [comment]
I would still do it, but not only after https://github.com/SwissDataScienceCenter/renku/pull/733 has been merged.
